packages/ui/src/components/va-pagination/VaPagination.vue

```diff
@@ -224,20 +216,45 @@ export default defineComponent({
       placeholder: `${currentValue.value}/${lastPage.value}`,
     }))
 
+    const paginationAttributesComputed = computed(() => ({
+      preset: props.preset,
+      plain: props.plain,
+      size: props.size,
+      color: props.color,
+      borderColor: props.borderColor,
+    }))
+
+    const inputClassComputed = useBem('va-pagination__input', () => ({
+      ...pick(props, ['plain']),
+    }))
+
+    const classComputed = useBem('va-pagination', () => ({
+      ...pick(props, ['gapped', 'rounded']),
+      bordered: !!props.borderColor,
+    }))
+
+    const currentPageButtonStyle = computed(() => ({
+      outline: props.currentPageBorderColor ? `1px solid ${getColor(props.currentPageBorderColor)}` : 'inherit',
+      outlineOffset: props.currentPageBorderColor ? '-1px' : 'inherit',
+    }))
```

Comment:
In this case `currentPageButton` will not have any style defined unless the user passes `currentPageBorderColor`.
Maybe it's better to have an `active` state for `va-button`.

Reply:
I agree, but still don't know how to implement this - props to slotted buttons are passed via common button group props object - can't specify props for separate button.